linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd

```diff
@@ -1269,6 +1269,246 @@ table_community_deaths_top_zones_time %>%
 
 
 
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+# Healthcare worker cases {.tabset .tabset-fade .tabset-pills}
+      
+<!-- ======================================================= -->
+## Outline 
+
+We use the variable `hcw` to estimate the proportions of cases
+who are healthcare workers:
+        
+* overall, monthly
+* by sub-coordination, monthly
+
+Each analysis will present:
+        
+* overall results aggregated for the whole time period
+* a temporal breakdown
+      
+For each proportion, we derive the exact binomial confidence interval.
+      
+      
+      
+<!-- ======================================================= -->
+## Note
+        
+We will rely on `adorn_totals` for computing totals. This approach needs to take into account an
+[issue](https://github.com/sfirke/janitor/issues/285) with `adorn_totals`, which
+ignores the first column of data assuming it is a grouping factor. This will not
+be a problem for further stratification, where the function can be used safely.
+      
+      
+      
+<!-- ======================================================= -->
+## Overall
+        
+The analysis proceeds with the following steps:
+        
+1. calculate overall numbers and corresponding proportions and confidence intervals
+      
+2. calculate monthly numbers and corresponding proportions and confidence intervals
+      
+3. plot temporal trends
+      
+4. create a table from 1, renaming columns and displaying it 
+      
+5. create a table from 2, renaming columns and displaying it 
+      
+      
+```{r hcw_overall}
+      
+## step 1
+hcw_overall <- x %>%
+  count(hcw) %>%
+  spread(hcw, n, fill = 0, drop = FALSE) %>%
+  mutate(total = nrow(x), # avoid adorn_totals
+        denom = total - unknown,
+        p = prop_to_perc(hcw / denom),
+        p_lower = prop_ci(hcw, denom, "lower", perc = TRUE),
+        p_upper = prop_ci(hcw, denom, "upper", perc = TRUE))
+      
+      
+## step 2
+hcw_overall_time <- x %>%
+  count(month_report, hcw) %>%
+  spread(hcw, n, fill = 0, drop = FALSE) %>%
+  adorn_totals(c("col"), name = "total") %>% 
+  mutate(denom = total - unknown,
+        p = prop_to_perc(hcw / denom),
+        p_lower = prop_ci(hcw, denom, "lower", perc = TRUE),
+        p_upper = prop_ci(hcw, denom, "upper", perc = TRUE))
+      
+      
+## step 3
+ggplot(hcw_overall_time, aes(x = month_report, y = p)) +
+        theme_bw() +
+        geom_ribbon(aes(ymin = p_lower, ymax = p_upper),
+              alpha = .3, fill = color_hcw) +
+        geom_point(color = color_hcw) +
+        geom_line(color = color_hcw) +
+        scale_y_continuous(limits = c(0, 25)) +

```

Comment:
avoid hard-coded limits for axes, as these may not be adapted in future updates (unless we have an objective fixed scale)

Reply:
I chose this in order to see the downward trend. HCWs should be a very low percentage.